src/controllers/search.controller.ts

```diff
@@ -25,13 +25,28 @@ const route = async (req: Request, res: Response) => {
 
     try {
       const companyResource: CompaniesResource = await getCompanies(companyName, cookies.get(SEARCH_WEB_COOKIE_NAME));
+      let topHit: string  = companyResource.topHit;
       searchResults = companyResource.results.map((result) => {
         const status = result.items.company_status;
         let capitalisedStatus: string = "";
         if (status !== undefined) {
           capitalisedStatus = status.charAt(0).toUpperCase() + status.slice(1);
         }
 
+        if (result.items.corporate_name === topHit) {
+          return [
+            {
+              html: `<a id="centre-page" href="${result.links.self}">${result.items.corporate_name}</a>`,
+              classes: "nearest"
+            },
+            {
+              text: result.items.company_number,
+            },
+            {
+              text: capitalisedStatus,
+            },
+          ];
+        }
```

Comment:
The the text attributes are repeated in both returns. I would do something like this:
`let html = `<a href="${result.links.self}">${result.items.corporate_name}</a>`;
let classes = "";
if (result.items.corporate_name === topHit) {
    html =`<a id="centre-page" href="${result.links.self}">${result.items.corporate_name}</a>`,
    classes="nearest"
}

return [
    {
        html,
        classes
    },
    {
        text: result.items.company_number,
    },
    {
        text: capitalisedStatus,
    },
];`

Reply:
Thats a good shout, wasn't sure how to best do this but that definitely looks more concise and maintainable!

Reply:
Cool, the change you just made, is the html supposed to different. In the previous commit the top hit had an id of "centre page". ALso I realised the code I suggested didn't quite paste properly;
```
let html = `<a href="${result.links.self}">${result.items.corporate_name}</a>`;
let classes = "";
if (result.items.corporate_name === topHit) {
    html =`<a id="centre-page" href="${result.links.self}">${result.items.corporate_name}</a>`,
    classes="nearest"
}

return [
    {
        html,
        classes
    },
    {
        text: result.items.company_number,
    },
    {
        text: capitalisedStatus,
    },
];
```

Reply:
Yeah sorry I noticed that I wasn't using the "centre-page" ID it was a leftover when I was testing it

Reply:
Alright cool, looks fine to me